core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -27,15 +27,17 @@ object WorkflowSubmissionActor {
             executionServiceCluster: ExecutionServiceCluster,
             batchSize: Int,
             credential: Credential,
+            processInterval: FiniteDuration,
             pollInterval: FiniteDuration,
             maxActiveWorkflowsTotal: Int,
             maxActiveWorkflowsPerUser: Int,
             runtimeOptions: Option[JsValue]): Props = {
-    Props(new WorkflowSubmissionActor(dataSource, methodRepoDAO, googleServicesDAO, executionServiceCluster, batchSize, credential, pollInterval, maxActiveWorkflowsTotal, maxActiveWorkflowsPerUser, runtimeOptions))
+    Props(new WorkflowSubmissionActor(dataSource, methodRepoDAO, googleServicesDAO, executionServiceCluster, batchSize, credential, processInterval, pollInterval, maxActiveWorkflowsTotal, maxActiveWorkflowsPerUser, runtimeOptions))
   }
 
   sealed trait WorkflowSubmissionMessage
-  case object ScheduleNextWorkflowQuery extends WorkflowSubmissionMessage
```

Comment:
imo these are two different commands and how long to wait is down to the receiver of the message. also the bike shed should be painted yellow